content/blog/2019/smashing.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Smashing CSS Follow-Up Q&nbsp;&&nbsp;A
+title: Smashing CSS Follow-Up Q&A
```

Comment:
Didn't like my hack for getting a fancy ampersand?

Reply:
I mean, I liked the result, but it does feel like a hack, and we're not using it consistently. Elsewhere we use `Q&A`. I guess I'd vote for a more robust and consistent solution?

Reply:
I agree. Doesn't seem high priority, tho.

Reply:
Yeah, agreed. The reason I discovered it is because the RSS validator complained about the HTML escaped entity in the `<title>` attr. Right now we assume titles are text, and only escape the contents.